lib/nx_gzlib.c

```diff
@@ -185,14 +203,33 @@ gzFile nx_gzopen_(const char* path, int fd, const char *mode)
 	return file;
 }
 
-int nx_gzread(gzFile file, void *buf, unsigned len)
+gzFile nx_gzopen_(const char* path, int fd, const char *mode) {
```

Comment:
We don't need `nx_gzopen_` anymore, it got replaced by `__gzopen_` in this commit.

Reply:
Done.